From b2e57082cb30e0810ab1e89b5ba32e635f3bbc5d Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Tue, 4 Mar 2025 09:26:44 +0100
Subject: [PATCH] add pthread

---
 src/Makevars.win | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makevars.win b/src/Makevars.win
index 5a6fd9a..321deb0 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -11,7 +11,7 @@ else
 RWINLIB = ../windows/poppler
 PKG_CPPFLAGS = -I$(RWINLIB)/include/poppler/cpp -I$(RWINLIB)/include/poppler
 PKG_LIBS = -L$(RWINLIB)/lib${subst gcc,,${COMPILED_BY}}${R_ARCH} -L$(RWINLIB)/lib \
-	-lpoppler-cpp -lpoppler -llcms2 -ljpeg -lpng16 -ltiff -lopenjp2 -lfreetype -lfreetype -lbz2 -liconv -lz
+	-lpoppler-cpp -lpoppler -llcms2 -ljpeg -lpng16 -ltiff -lopenjp2 -lfreetype -lfreetype -lbz2 -liconv -lz -pthread
 POPPLERDATA = $(RWINLIB)/share/poppler
 endif

